release-notes/opensearch-documentation-release-notes-2.12.0.md

```diff
@@ -0,0 +1,42 @@
+- Add warning related to anomaly detection and concurrent segment search [#6411](https://github.com/opensearch-project/documentation-website/pull/6411)
+- Adds metrics framework documentation [#6393](https://github.com/opensearch-project/documentation-website/pull/6393)
+- Add documentation for new reranking feature in 2.12 [#6368](https://github.com/opensearch-project/documentation-website/pull/6368)
+- Add documentation for saved object management to support multiple datasource [#6364](https://github.com/opensearch-project/documentation-website/pull/6364)
+- Add things to know for concurrent segment search [#6362](https://github.com/opensearch-project/documentation-website/pull/6362)
+- Add cross-encoder model documentation [#6357](https://github.com/opensearch-project/documentation-website/pull/6357)
+- Concurrent segment search GA and API changes [#6356](https://github.com/opensearch-project/documentation-website/pull/6356)
+- Improve terms aggregation documentation and add concurrent segment search caveats [#6355](https://github.com/opensearch-project/documentation-website/pull/6355)
+- Updating documentation for new datetime format [#6337](https://github.com/opensearch-project/documentation-website/pull/6337)
+- Add documentation for cluster-level coordination setting changes [#6336](https://github.com/opensearch-project/documentation-website/pull/6336)
+- Update remote cluster documentation for global metadata and new dynamic settings [#6309](https://github.com/opensearch-project/documentation-website/pull/6309)
+- Add include_named_queries_score to docs [#6306](https://github.com/opensearch-project/documentation-website/pull/6306)
+- Add remove_by_pattern ingest processor [#6295](https://github.com/opensearch-project/documentation-website/pull/6295)
+- Add copy ingest processor [#6294](https://github.com/opensearch-project/documentation-website/pull/6294)
+- Add alias support to Security Analytics [#6284](https://github.com/opensearch-project/documentation-website/pull/6284)
+- Update SAML tutorial with new default Admin settings [#6279](https://github.com/opensearch-project/documentation-website/pull/6279)
+- Add threat intelligence for 2.12 [#6273](https://github.com/opensearch-project/documentation-website/pull/6273)
+- Added documentation for changing the default values of HNSW hyper parameters in k-NN plugin [#6272](https://github.com/opensearch-project/documentation-website/pull/6272)
+- Document new experimental HTTP transport based on Netty 4 and Project Reactor [#6265](https://github.com/opensearch-project/documentation-website/pull/6265)
+- Add Query Insights documentation [#6261](https://github.com/opensearch-project/documentation-website/pull/6261)
+- Add flow framework documentation [#6257](https://github.com/opensearch-project/documentation-website/pull/6257)
+- [DOC] Added documentation for admission control stats [#6253](https://github.com/opensearch-project/documentation-website/pull/6253)
```

Comment:
Global nit, should we edit these titles so that they have the same format? Not a requirement, but would make these slightly more readable.

```suggestion
- [DOC] Add documentation for admission control stats [#6253](https://github.com/opensearch-project/documentation-website/pull/6253)
```

Reply:
Ideally, people should follow the PR naming guidelines. We should enforce this at doc review time.